src/reload/client.ts

```diff
@@ -0,0 +1,31 @@
+let needToReload = false;
+
+export default function reloadOnUpdated() {
+  // reload
+  function reload() {
+    window.location.reload();
+    socket.send("remove");
+  }
+
+  // reload when document visible
+  document.addEventListener("visibilitychange", () => {
+    !document.hidden && needToReload && reload();
+  });
+
+  // reload server setting
+  const socket = new WebSocket("ws://localhost:8081");
+  socket.addEventListener("open", () => {
+    socket.send("add");
+  });
+  socket.addEventListener("message", (event) => {
+    if (event.data !== "update") {
+      return;
+    }
+    // disable reload when document hidden
+    if (document.hidden) {
```

Comment:
why action reload when document.hidden is true?

Reply:
In situations where a tab is hidden (when you are viewing another tab), I didn't think it was necessary to refresh that tab. And the intention was to refresh the page to show the updated page as soon as the user sees the tab again.

Reply:
By changing needToReload to true, the next tab will be refreshed when it is shown.

Reply:
Oooh... I'm missing the code to change the need To Reload back to false

https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite/pull/25/commits/e90032c744f5a94f0653be10ea420438a6f5fbf4